test/unit/solver/mc/test_triangular.cpp

```diff
@@ -26,15 +27,30 @@ using namespace dlaf::matrix::test;
 using namespace dlaf_test;
 using namespace testing;
 
+::testing::Environment* const comm_grids_env =
+    ::testing::AddGlobalTestEnvironment(new CommunicatorGrid6RanksEnvironment);
+
 template <typename Type>
 class TriangularSolverLocalTest : public ::testing::Test {};
-
 TYPED_TEST_SUITE(TriangularSolverLocalTest, MatrixElementTypes);
 
+template <typename Type>
+class TriangularSolverDistributedTest : public ::testing::Test {
+public:
+  const std::vector<CommunicatorGrid>& commGrids() {
+    return comm_grids;
+  }
+};
+TYPED_TEST_SUITE(TriangularSolverDistributedTest, MatrixElementTypes);
+
 std::vector<blas::Diag> blas_diags({blas::Diag::NonUnit, blas::Diag::Unit});
-std::vector<blas::Op> blas_ops({blas::Op::NoTrans, blas::Op::Trans, blas::Op::ConjTrans});
-std::vector<blas::Side> blas_sides({blas::Side::Left, blas::Side::Right});
-std::vector<blas::Uplo> blas_uplos({blas::Uplo::Lower, blas::Uplo::Upper});
+// std::vector<blas::Op> blas_ops({blas::Op::NoTrans, blas::Op::Trans, blas::Op::ConjTrans});
+// std::vector<blas::Side> blas_sides({blas::Side::Left, blas::Side::Right});
+// std::vector<blas::Uplo> blas_uplos({blas::Uplo::Lower, blas::Uplo::Upper});
```

Comment:
In this way you also disable triangular solver local tests. (See comment below for a possible solution).

Reply:
You're right..